Applications/IoTBlock-test/main.cpp

```diff
@@ -87,11 +106,12 @@ int main()
         MAIN_ERROR("Failed to configure Shard Edge");
     }
 
-    int err = node.initCommunication(&can_layer);
-    if(err != VITROIO_ERR_SUCCESS){
+    while(node.initCommunication(&can_layer) != VITROIO_ERR_SUCCESS){
```

Comment:
@Stojek139808 was that not added in latest SDK release? Are those changes rebased?

Reply:
Cleared out commits to make it more clear. I suppose it was my bad that this wasn't commited before.
Didn't notice because now shards initialize correctly, whereas before they would do it quite often.